app-mobilephone/gammu/gammu-1.38.4.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils cmake-utils
+
+DESCRIPTION="A tool to handle your cellular phone"
+HOMEPAGE="http://wammu.eu/gammu/"
+SRC_URI="http://dl.cihar.com/${PN}/releases/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="bluetooth curl dbi debug irda mysql nls postgres usb odbc"
+
+COMMON_DEPEND="
+	dev-libs/glib:2=
+	virtual/libgudev:=
+	bluetooth? ( net-wireless/bluez:= )
+	curl? ( net-misc/curl:= )
+	dbi? ( >=dev-db/libdbi-0.8.3:= )
+	mysql? ( virtual/mysql:= )
+	postgres? ( dev-db/postgresql:= )
+	usb? ( virtual/libusb:1= )
+"
+DEPEND="
+	${COMMON_DEPEND}
+	irda? ( virtual/os-headers )
+	odbc? ( dev-db/unixODBC )
+	nls? ( sys-devel/gettext )
+"
+RDEPEND="
+	${COMMON_DEPEND}
+	dev-util/dialog
+	virtual/libiconv
+"
+
+# sys-devel/gettext is needed for creating .mo files
+# Supported languages and translated documentation
+# Be sure all languages are prefixed with a single space!
+MY_AVAILABLE_LINGUAS=" af ar bg bn ca cs da de el en_GB es et fi fr gl he hu id it ko nl pl pt_BR ro ru sk sv sw tr zh_CN zh_TW"
+IUSE="${IUSE} ${MY_AVAILABLE_LINGUAS// / linguas_}"
```

Comment:
`LINGUAS` is deprecated because it breaks EAPI 5+ USE_EXPAND logic. In this particular case, I don't think stripping l10ns is worth the hassle, so please just remove all the relevant logic.